src/client/containers/ProfilePage/ProfilePage.Style.css

```diff
@@ -2,6 +2,10 @@
   display: flex;
 }
 
+.profilepage h1 {
```

Comment:
Yet another pattern for naming classes... 🙂 
It's normally a good idea to stick with 1 naming pattern.